completion/available/rvm.completion.bash

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Bash completion support for  RVM.
+# Source: https://rvm.io/workflow/completion
+
+
+_rvm_complete() {
```

Comment:
I think you need to get rid of this function. From what I understand, you only really need to have the next line in this file.